### PR TITLE
EIP-6187 Relax email and street

### PR DESCRIPTION
**Pull request by vm-valtech**
https://technologyprogramme.atlassian.net/browse/EIP1-6187

### CODE REVIEW

File: src/test/kotlin/uk/gov/dluhc/registercheckerapi/messaging/InitiateRegisterCheckMessageListenerIntegrationTest.kt [hunk 1]
```diff
@@ -29,7 +31,11 @@ internal class InitiateRegisterCheckMessageListenerIntegrationTest : Integration
     @Test
     fun `should process message received on queue`() {
         // Given
-        val message = buildInitiateRegisterCheckMessage()
+        val message = buildInitiateRegisterCheckMessage(
+            personalDetail = buildRegisterCheckPersonalDetail(
+                address = buildRegisterCheckAddress(street = "")
```
Comment (gurpiarbassi) [inline src/test/kotlin/uk/gov/dluhc/registercheckerapi/messaging/InitiateRegisterCheckMessageListenerIntegrationTest.kt:36]: don't think we need to change this integration test, otherwise this test becomes specific to empty street

Comment (kristerbone) [reply src/test/kotlin/uk/gov/dluhc/registercheckerapi/messaging/InitiateRegisterCheckMessageListenerIntegrationTest.kt]: 2 tests? 1 with, 1 without?

**Review by gurpiarbassi — APPROVED** [review]
